ocs_ci/helpers/dr_helpers_ui.py

```diff
@@ -641,4 +639,88 @@ def check_apps_running_on_selected_cluster(
+        log.info("Clear existing text from search bar if any")
+        acm_obj.do_clear(acm_loc["search-bar"])
+        log.info("Enter the workload to be searched")
+        acm_obj.do_send_keys(acm_loc["search-bar"], text=workload_to_check)
+        action_status = acm_obj.wait_until_expected_text_is_found(
+            locator=acm_loc["no-results-found"],
+            expected_text="No results found",
+            timeout=timeout,
+        )
+        if action_status:
+            fetch_status = acm_obj.get_element_text(locator=acm_loc["no-results-found"])
+            assert action_status, "Application present in UI"
+            log.info(
+                f"Successfully verified on ACM UI, Application Not Found, fetch status is {fetch_status}"
+            )
+            return True
+        else:
+            log.error("One or more matches found, Incorrect workload name")
+            return False
+
+
+def delete_application_ui(acm_obj, workload_to_delete, timeout=60):
+    """
+    Function to delete specified workloads on ACM UI
+
+    Args:
+        acm_obj (AcmAddClusters): ACM Page Navigator Class
+        workload_to_delete (str): Specify the workload to delete, otherwise none will be deleted
+        timeout (int): timeout to wait for certain elements to be found on the ACM UI
+
+    Returns:
+        bool: True if the application is deleted successfully, false otherwise
+
+    """
+    log.info(f"workload_to_delete {workload_to_delete}")
+    if not verify_application_not_present_in_ui(
+        acm_obj, workload_to_check=workload_to_delete, timeout=60
+    ):
+        ocp_version = get_ocp_version()
+        acm_loc = locators[ocp_version]["acm_page"]
+        acm_obj.navigate_applications_page()
+        log.info("Click on search bar")
+        acm_obj.do_click(acm_loc["search-bar"])
+        log.info("Clear existing text from search bar if any")
+        acm_obj.do_clear(acm_loc["search-bar"])
+        log.info("Enter the workload to be searched")
+        acm_obj.do_send_keys(acm_loc["search-bar"], text=workload_to_delete)
+        log.info("Click on kebab menu option")
+        acm_obj.do_click(acm_loc["kebab-action"], enable_screenshot=True)
+        acm_obj.do_click(acm_loc["delete-app"], enable_screenshot=True, timeout=timeout)
+        if not acm_obj.get_checkbox_status(acm_loc["remove-app-resources"]):
+            acm_obj.select_checkbox_status(
+                status=True, locator=acm_loc["remove-app-resources"]
+            )
+        acm_obj.do_click(acm_loc["delete"], enable_screenshot=True, timeout=timeout)
+        # Check if the workload got deleted
+        assert verify_application_not_present_in_ui(
+            acm_obj, workload_to_check=workload_to_delete, timeout=60
+        ), f"Application {workload_to_delete} still exists"
+        log.info(f"Application {workload_to_delete} got deleted successfully")
+        return True
+    else:
+        log.error("Application not present to delete")
```

Comment:
Is this is right error message?

Reply:
@Shrivaibavi Could you pls respond to this query?

Reply:
What if we have multiple apps with the same name and it fails the test? That's why I suggested to apply the filter for the type of workload in one of my other comments.

Reply:
The main intention of the PR is to deploy apps on both the clusters and failover multiple apps from C1 to C2 and slowly got into delete app from UI as teardown. 

Its a very nice suggestion, But can only be incorporated in the upcoming PR. I believe there is no point in holding this PR for the above small correction. Please add your comment in this open issue https://github.com/red-hat-storage/ocs-ci/issues/10365

Reply:
ack